src/features/jserrors/aggregate/index.js

```diff
@@ -206,6 +206,11 @@ export class Aggregate extends AggregateBase {
     // still send EE events for other features such as above, but stop this one from aggregating internal data
     if (this.blocked) return
 
+    if (err.__newrelic) {
```

Comment:
Is it possible that this will be a breaking point for multiple agents running on the same page?